apps/ehr/src/features/visits/in-person/layout/InPersonLayout.tsx

```diff
@@ -38,6 +38,7 @@ export const InPersonLayout: React.FC = () => {
   const [recordingAnchorElemement, setRecordingAnchorElement] = React.useState<HTMLButtonElement | null>(null);
   const recordingElementID = 'recording-element';
   const recordingOpen = Boolean(recordingAnchorElemement);
+  const isFollowup = encounter ? isFollowupEncounter(encounter) : false;
```

Comment:
Nit: Perhaps it would be more convenient to use a value from the navigation context. Possibly in the future, there will be additional roles that will work with follow-up visits, so it makes sense to add a separate visit type field to the navigation context, so that it can always be retrieved directly without calculation.